### ISSUE TITLE
Error during installation: package manager could not prepare updates

### ISSUE THREAD

**Issue by spsf64** [CachyOS/cachyos-calamares#6]
Tried many times, similar error; ext4 + luks + XFCE + printing support

Error: 

The package manager could not prepare updates. The command  pre>pacman</pre> returned error code 1

error log: https://termbin.com/pusj

**Comment by ptr1337** [comment]
> Tried many times, similar error, XFCE + printing support
> 
> The package manager could not prepare updates. The command
> 
> pacman
> returned error code 1
> error log: https://termbin.com/pusj

https://build.cachyos.org/ISO/testing/

Please use this ISO.
A offical Release will come tomorrow

**Comment by vnepogodin** [comment]
@spsf64 
that's due to our key expired
reboot and run before launching installer:

```
sudo pacman-key --recv-keys F3B607488DB35A47 --keyserver keyserver.ubuntu.com
sudo pacman-key --lsign-key F3B607488DB35A47
```

or just use offline install that also works, but requires doing same step after

will release ISO soon with the new keyrings

**Comment by spsf64** [comment]
@ptr1337 @vnepogodin , thanks for the prompt reply, will try the new ISO and report back!
Forgot to add, I used ventoy to boot, any restriction?

**Comment by spsf64** [comment]
Ok, tested the new ISO, installation finished without any problem, but disk won't boot. Does not even show up in bios/uefi
I tried to install to a 128GB (/sda) + refind + luks + xfce
This is the lsblk -f from my booted arch nvme

linux@archlinux ~> lsblk -f
NAME        FSTYPE      FSVER LABEL     UUID                                 FSAVAIL FSUSE% MOUNTPOINTS
sda                                                                                         
├─sda1      vfat        FAT32           49F5-5B61                                           
└─sda2      crypto_LUKS 1               a5b2838c-01fe-4bb7-aec2-6c09f095927b                
sr0                                                                                         
nvme0n1                                                                                     
├─nvme0n1p1 vfat        FAT32           3EF9-B75E                             418.7M    18% /boot
├─nvme0n1p2 ntfs              Win11     5634FD0534FCE93F                                    
├─nvme0n1p3                                                                                 
└─nvme0n1p4 crypto_LUKS 2               569bb707-c2a4-45ff-8ff8-d3c80xxxxxxx                
  └─luksdev ext4        1.0   ArchNvme4 4b008b54-d345-4bc1-acb4-e15f4xxxxxxx  108.8G    50% /
linux@archlinux ~>

**Comment by ptr1337** [comment]
Please try to not use encryption with refind or use systemd-boot.
Paritioning seems fine from your side.

I need to test refind encryption. systemd-boot works fine

**Comment by spsf64** [comment]
Thanks, will check again soon. I was trying to test refind...

**Comment by mpact1984** [comment]
I have the same issue here. 
I coose: Autopartition with xfs, wayfire, Kernel linux-cachyos-bore, no additional packages

 Installer quit with an error message `'The command <pre>pacman</pre> returned error code 1.'` 

I tried your workaround with rebooting the live system, adding the new pacman keys and afterwards installing the system, but looks like it's not working for me.

https://termbin.com/1iff
https://termbin.com/apym

**Comment by ptr1337** [comment]
> @ptr1337 @vnepogodin , thanks for the prompt reply, will try the new ISO and report back! Forgot to add, I used ventoy to boot, any restriction?

Ventoy does not problems

**Comment by spsf64** [comment]
> > @ptr1337 @vnepogodin , thanks for the prompt reply, will try the new ISO and report back! Forgot to add, I used ventoy to boot, any restriction?
> 
> Ventoy does not problems

I think you can close this issue as the ISO worked properly, maybe put a note in download section about the use of "refind + encryption" problem.
Thank you

**Comment by ptr1337** [comment]
> > > @ptr1337 @vnepogodin , thanks for the prompt reply, will try the new ISO and report back! Forgot to add, I used ventoy to boot, any restriction?
> > 
> > 
> > Ventoy does not problems
> 
> I think you can close this issue as the ISO worked properly, maybe put a note in download section about the use of "refind + encryption" problem. Thank you

Since someone reported once that encryption worked, we kept it enabled.
But we can just disable it for the next release :)

**Comment by spsf64** [comment]
You mean, disable refind? Or disable encryption?
Let me try again a new install with systemd-boot+encryption... Will report back

**Comment by ptr1337** [comment]
> You mean, disable refind? Or disable encryption? Let me try again a new install with systemd-boot+encryption... Will report back

Disabling encryption just for refind. We can disable it in the refind config :)

systemd-boot encryption should work fine.

**Comment by spsf64** [comment]
> Disabling encryption just for refind. We can disable it in the refind config :)
> 
> systemd-boot encryption should work fine.

Ok, good idea. 

I just did an install to a external usb flash drive with systemd-boot+luks(ext4)+xfce and everything worked fine, it is very fast, portable and secure!
Thanks for your help.
